packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -6725,3 +6769,28 @@ class _EditableTextTapUpOutsideAction extends ContextAction<EditableTextTapUpOut
     // The default action is a no-op.
   }
 }
+
+/// A [TextSpan] that overrides the style of its children with a given
+/// [TextStyle].
+class _OverridingTextStyleTextSpan extends TextSpan {
```

Comment:
Should we add a comment to keep this up-to-date with the other `_OverridingTextStyleTextSpan`? I don't feel strongly either ways, up to you!